tests/sentry/tasks/test_derive_code_mappings.py

```diff
@@ -81,8 +88,56 @@ def test_derive_code_mappings(self):
             derive_code_mappings(self.project.id, self.event_data)
             code_mapping = RepositoryProjectPathConfig.objects.all()[0]
             # ./app/foo.tsx -> foo.tsx -> static/app/foo.tsx
```

Comment:
```suggestion
            # ./app/foo.tsx -> app/foo.tsx -> static/app/foo.tsx
```

The stack root drops "./", thus, `app/foo.tsx`